adcaptcha.php

```diff
@@ -33,10 +34,11 @@
 require_once plugin_dir_path(__FILE__) . 'src/Plugin/Woocommerce/Registration.php';
 require_once plugin_dir_path(__FILE__) . 'src/Plugin/ContactForm7/Forms.php';
 require_once plugin_dir_path(__FILE__) . 'src/Plugin/Mailchimp/Forms.php';
+require_once plugin_dir_path(__FILE__) . 'src/Plugin/NinjaForms/Forms.php';
 
 use AdCaptcha\Instantiate;
 
-const PLUGIN_VERSION = '1.0.1';
+const PLUGIN_VERSION_adCAPTCHA = '1.1.0';
```

Comment:
This is an odd capitalisation. Should be this:

```suggestion
const PLUGIN_VERSION_ADCAPTCHA = '1.1.0';
```